<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="RESULT1_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Result2.计算过程：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getPicBtn">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>210</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>查看路径图</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="STEP1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Step1.请输入标准集目录：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="pictureSlct">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="getPathBtn">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>50</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>读取标准集</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>671</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="testSetSlct">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="STEP2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Step2.选择标准集并计算：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pathEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startCltBtn">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>130</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>开始演化计算</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="RESULT1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Result1.查看大幅优化解的路径图：</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
